atom/browser/browser_win.cc

```diff
@@ -54,14 +56,20 @@ bool GetProtocolLaunchPath(mate::Arguments* args, base::string16* exe) {
     *exe = path.value();
   }
 
-  // Read in optional args arg
-  std::vector<base::string16> launch_args;
-  if (args->GetNext(&launch_args) && !launch_args.empty())
-    *exe = base::StringPrintf(L"\"%s\" %s \"%%1\"",
+  if (launch_args.empty()) {
+    base::string16 formatString = includeOriginalArg ?
+      L"\"%s\" \"%%1\"" :
+      L"\"%s\"";
```

Comment:
@CharlieHess is there a reason the path needs to be quoted here for login item settings path?

I ask because I'm wondering if this breaks backwards compatibility for people that previously used `setLoginItemSettings` which did not quote the path value in the old implementation. After this change, if they called `getLoginItemSettings` the paths would not match because it is not expected the registry value to be quoted when in previous releases it did not quote it.

Thoughts?